src/libraries/System.Memory/ref/System.Memory.cs

```diff
@@ -283,6 +283,7 @@ public static partial class SequenceReaderExtensions
         public bool TryReadTo(out System.Buffers.ReadOnlySequence<T> sequence, System.ReadOnlySpan<T> delimiter, bool advancePastDelimiter = true) { throw null; }
         public bool TryReadTo(out System.Buffers.ReadOnlySequence<T> sequence, T delimiter, bool advancePastDelimiter = true) { throw null; }
         public bool TryReadTo(out System.Buffers.ReadOnlySequence<T> sequence, T delimiter, T delimiterEscape, bool advancePastDelimiter = true) { throw null; }
+        public bool TryReadTo(out System.ReadOnlySpan<T> sequence, System.ReadOnlySpan<T> delimiter, bool advancePastDelimiter = true) { throw null; }
```

Comment:
nit: the parameter name should be span instead of sequence. I'm not sure if docs or VS use this, so might not matter?

Reply:
We should be consistent with the other APIs

Reply:
Is correct in the implementation; is only the reference assembly that is incorrect. PR to resolve https://github.com/dotnet/runtime/pull/39594